pkg/internal/reporter/reporter_test.go

```diff
@@ -78,31 +78,31 @@ func TestReportShouldSuccessfullyReportResults(t *testing.T) {
-			TrafficGeneratorInErrorPackets:  expectedTrafficGeneratorInErrorPackets,
-			DPDKRxTestPackets:               expectedDPDKRxTestPackets,
-			DPDKPacketsRxDropped:            expectedDPDKPacketsRxDropped,
-			DPDKPacketsTxDropped:            expectedDPDKPacketsTxDropped,
-			DPDKVMNode:                      expectedDPDKVMNode,
-			TrafficGeneratorNode:            expectedTrafficGeneratorNode,
+			TrafficGenSentPackets:          expectedTrafficGenSentPackets,
+			TrafficGenOutputErrorPackets:   expectedTrafficGenOutputErrorPackets,
+			TrafficGeneratorInErrorPackets: expectedTrafficGeneratorInErrorPackets,
+			DPDKRxTestPackets:              expectedDPDKRxTestPackets,
+			DPDKPacketsRxDropped:           expectedDPDKPacketsRxDropped,
+			DPDKPacketsTxDropped:           expectedDPDKPacketsTxDropped,
+			DPDKVMNode:                     expectedDPDKVMNode,
+			TrafficGeneratorNode:           expectedTrafficGeneratorNode,
 		}
 
 		assert.NoError(t, testReporter.Report(checkupStatus))
 
 		expectedReportData := map[string]string{
-			"status.succeeded":                                 strconv.FormatBool(true),
-			"status.failureReason":                             "",
-			"status.startTimestamp":                            timestamp(checkupStatus.StartTimestamp),
-			"status.completionTimestamp":                       timestamp(checkupStatus.CompletionTimestamp),
-			"status.result.trafficGenSentPackets":              fmt.Sprintf("%d", checkupStatus.Results.TrafficGenSentPackets),
-			"status.result.trafficGeneratorOutputErrorPackets": fmt.Sprintf("%d", checkupStatus.Results.TrafficGeneratorOutErrorPackets),
-			"status.result.trafficGeneratorInErrorPackets":     fmt.Sprintf("%d", checkupStatus.Results.TrafficGeneratorOutErrorPackets),
-			"status.result.DPDKRxTestPackets":                  fmt.Sprintf("%d", checkupStatus.Results.DPDKRxTestPackets),
-			"status.result.DPDKRxPacketDrops":                  fmt.Sprintf("%d", checkupStatus.Results.DPDKPacketsRxDropped),
-			"status.result.DPDKTxPacketDrops":                  fmt.Sprintf("%d", checkupStatus.Results.DPDKPacketsTxDropped),
-			"status.result.trafficGeneratorNode":               checkupStatus.Results.TrafficGeneratorNode,
-			"status.result.DPDKVMNode":                         checkupStatus.Results.DPDKVMNode,
+			"status.succeeded":                             strconv.FormatBool(true),
+			"status.failureReason":                         "",
+			"status.startTimestamp":                        timestamp(checkupStatus.StartTimestamp),
+			"status.completionTimestamp":                   timestamp(checkupStatus.CompletionTimestamp),
+			"status.result.trafficGenSentPackets":          fmt.Sprintf("%d", checkupStatus.Results.TrafficGenSentPackets),
+			"status.result.trafficGenOutputErrorPackets":   fmt.Sprintf("%d", checkupStatus.Results.TrafficGenOutputErrorPackets),
+			"status.result.trafficGeneratorInErrorPackets": fmt.Sprintf("%d", checkupStatus.Results.TrafficGenOutputErrorPackets),
```

Comment:
This line should not be changed by this commit.

Reply:
Good catch!